### ISSUE TITLE
Checking a tool for the subscription 

### ISSUE THREAD

**Issue by EmanHylooz** [EmanHylooz/Abjjad-Product-Marketing#43]
This tool was recommended by Muhannad (ShopGo) as they worked on a subscription model and it is like a dashboard that can replace the excel sheet that we need to fill on weekly basis. 
No we can of course setup our data using excel, to figure out what data we need to capture and where from. This tool can be an advance way of actually presenting this data that will help us in data analysis. 

The tool is: https://chartmogul.com/

Please check it out and tell me your thoughts about it.

**Comment by Farahalfayez** [comment]
@EmanHylooz Can you take a look at the integration options and tell me if we have any of the systems? If not what's our best way to integrate? https://chartmogul.com/integrations/ 

It's a tool that I would like to see in action so I created an account and now on the free trial for 25 days. But need to integrate to explore what it offers.

**Comment by EmanHylooz** [comment]
Hi Farah,
Among all what we have we only have integration with PayPal which is not
used that much by our users. I guess our best bet for now is to do it the
manual way (excel sheets), and then we can explore the option of creating
our own APIs that will read data directly from the server (or the database)
and start filling from there.
I am not fully aware of the tool, but what I understood is that people
start filling stuff manually first (or through excel sheets), and then we
can figure out a way to connect with the server for some numbers where they
can be filled automatically.

Makes sense?

Best,

Eman Hylooz

Founder & CEO | Abjjad

Jordan: +962.79.5402965 | Lebanon: +961.81.667918 | *www.abjjad.com*
<http://www.abjjad.com/>

**Comment by Farahalfayez** [comment]
Yes makes sense. I will start the Google sheet integration and ask you for the needed data then.

**Comment by Farahalfayez** [comment]
@EmanHylooz

**Comment by EmanHylooz** [comment]
Perfect :)

Best,

Eman Hylooz

Founder & CEO | Abjjad

Jordan: +962.79.5402965 | Lebanon: +961.81.667918 | *www.abjjad.com*
<http://www.abjjad.com/>
